packages/rocketchat-i18n/i18n/en.i18n.json

```diff
@@ -2043,8 +2043,8 @@
   "How_responsive_was_the_chat_agent": "How responsive was the chat agent?",
   "How_satisfied_were_you_with_this_chat": "How satisfied were you with this chat?",
   "How_to_handle_open_sessions_when_agent_goes_offline": "How to Handle Open Sessions When Agent Goes Offline",
-  "I_ll_do_it_later": "I'll do it later",
-  "I_saved_my_password_close_this_message": "I saved my password, close this message",
+  "I_ll_do_it_later": "Do It Later",
```

Comment:
As we talked in private, I think that when translations change drasticaly like this, we should change the key also, in order to create a demand for new translations for other languages.